scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -39,47 +39,67 @@ fi
 L_PWD=$(pwd)
 cd "${DEVON_IDE_HOME}" || exit 255
 
-# Install Eclipse plugins by feature group id
-# $1: plugin feature group ID
-# $2: plugin URL (update site)
-function doInstallEclipsePlugin() {
-  # If plugin_name argument ${3} is passed
-  if [ -n "${3}" ]
+function doCreateProperties() {
+  #create properties file at propertiesPath with propertiesFile as name, if plugin name was not specified, use plugin_id as filename, file contents are plugin_url and plugin_id
+  local propertiesPath="${DEVON_IDE_HOME}/settings/eclipse/plugins"
+  local pluginId="${1}"
+  local pluginUrl="${2}"
+  local pluginName="${3}"
+  local pluginStatus="${4}"
+  if [ -z "${pluginStatus}" ]
   then
-    echo "Installing Eclipse plugin '${3}' from URL ${2}"
-  else
-    echo "Installing Eclipse plugin '${1}' from URL ${2}"
+    pluginStatus="false"
   fi
-
-  if [ -z "${1}" ]
+  local propertiesFile="${propertiesPath}/${pluginName}.properties"
+  if [ -z "${pluginName}" ]
+  then
+    pluginName="${pluginId}"
+    propertiesFile="${propertiesPath}/${pluginId}.properties"
+  fi
+  if [ -f "${propertiesFile}" ]
   then
-    doEcho "No plugins to install."
     return
   fi
+  echo -e "plugin_url=${pluginUrl}\nplugin_id=${pluginId}\nplugin_active=${pluginStatus}" >>"${propertiesFile}"
+}
+
+
+function doInstallEclipsePlugin() {
+  local doinstall_Plugin_Id="${1}"
+  local doinstall_Plugin_Url="${2}" #Update site
+  local doinstall_Plugin_Name="${3}" # if specified, the plugin will be installed and a properties file will be created with the name specified
```

Comment:
This comment is IMHO confusing as this function does not create such properties file. 
Feel free to move this hint to `doAddPlugin` as this is where the properties file gets created if `$3` is given.
BTW: why not just naming the variables short like `plugin_id` instead of `doinstall_Plugin_Id`? Also typically either do camlCase or snake_case but do not mix both...

Reply:
The reason why I chose function specific names is that plugin_id is a global variabel if I am not wrong. So I did not want to cause any undesired behaviour by messing with the global variable.
Should I refactor all variables to snake_case then, because the other functions that were written are already in snake_case

Reply:
> The reason why I chose function specific names is that plugin_id is a global variabel if I am not wrong. So I did not want to cause any undesired behaviour by messing with the global variable.

The local variable would not influence the global one but it could indeed cause confusion naming both identical.